### ISSUE TITLE
slot component example from the docs doesn't work for me

### ISSUE THREAD

**Issue by codekiln** [wearebraid/vue-formulate#146]
**Describe the bug**
The docs have [a slot component example](https://vueformulate.com/guide/inputs/slots/#a-slot-component-example). When I try to reproduce that example, it doesn't work - specifically, the tooltip icon and the tooltip do not show. 

**To Reproduce**
See https://github.com/codekiln/vue-playground/tree/master/vue-formulate/formulate-custom-slot1, which is a vue-cli project which has been configured according to the instructions at https://vueformulate.com/guide/inputs/slots/#a-slot-component-example.

As you can see, in the Vue debugger, `tooltip=False` and the `tooltip` is located elsewhere.
![image](https://user-images.githubusercontent.com/140930/86409749-7eab8c00-bc87-11ea-9d67-891b820cb4db.png)
![image](https://user-images.githubusercontent.com/140930/86409776-8a974e00-bc87-11ea-801e-a4d38710303b.png)

**Reproduction**
I've reproduced it in a repo above - it would be helpful in these bug posts if you had a default formulate codepen we could fork to create our example, as there's more setup to get the library to work in that context. 

**Expected behavior**
![image](https://user-images.githubusercontent.com/140930/86410084-26c15500-bc88-11ea-80c6-638d6c428354.png)

**Device information:**
 - Device: MBP
 - OS: osX 10.14.6
 - Browser Chrome
 - Version Evergreen

**Comment by justin-schroeder** [comment]
Thanks for this high quality bug report and reproduction 🙏 It makes my limited time so much more effective.

You indeed did locate 2 significant typos in the docs, I've fixed them on the documentation (`slotProp` was `tooltop` instead of `tooltip` and the component was using `context.tooltip` instead of the `tooltip` prop). I've also submitted a pull request to your reproduction with the example fixed and working so you have it has a reference.

https://github.com/codekiln/vue-playground/pull/32

Thanks again!
